### DISCUSSION TITLE
How to upgrade to latest release

### DISCUSSION THREAD

**Discussion by NicWhitham** [userjack6880/Open-DMARC-Analyzer#94] [General]
Please can somebody let me know the correct way to upgrade to the latest release. I tried just copying over the updated files but it doesn't work and I had to restore the previous version. Many thanks

**Comment by userjack6880** [comment]
What version are you coming from? The biggest change will be the config file.

**Reply by NicWhitham** [reply]
currently on version 1-β2 - looking to upgrade to latest release.  I retained the old config file when I tried to upgrade but couldn't see any differences.

**Comment by NicWhitham** [comment]
Any idea when I may expect a reply?

**Comment by userjack6880** [comment]
Sorry, I've been swamped for the past few days.

the following lines have been added since version 1 beta 2 and the final release of version 1

```
define('DB_TYPE', 'mysql'); // supported mysql and pgsql
```

If you're going to the version 2 alpha, add:

```
define('REPORT_TYPE', 'dmarc'); // supported dmarc, tls, all
```
